.github/workflows/rename_template_project_pr.yml

```diff
@@ -0,0 +1,24 @@
+name: rename-project-create-pr
```

Comment:
This can be deleted, it's no longer a template app

Reply:
removed